src/phone-input/country-select.js

```diff
@@ -186,7 +185,7 @@ export default function CountrySelect(props: CountrySelectPropsT) {
   );
 
   return (
-    <Block display="flex" alignItems="center">
+    <div style={{display: 'flex', alignItems: 'center'}}>
```

Comment:
Styled shouldn't be inlined. Could you create a styled component?

Reply:
updated